### ISSUE TITLE
Add a Stale bot for the issues

### ISSUE THREAD

**Issue by raphaelauv** [tiangolo/fastapi#2049]
There is a lot of issues ( almost 300 ) and a lot of them are just question of how use fastAPI and not real current bugs. 

I like to use the issues on github to know the current bugs on the latest version of a project. But here is complicated to have an overview of the "situation".

We could add a github bot [Stale](https://github.com/marketplace/stale) to manage and close old very inactive issues.

What do you think about it ? :)

**Comment by Kludex** [comment]
Looks nice :tada: 
Just a remark, don't we have some bot already running when @tiangolo answers something? I think if no comments are added after his in some days/weeks the issue is closed, but I'm not sure tho

**Comment by ArcLightSlavik** [comment]
There might be one but it does not seem to be running.
<https://github.com/tiangolo/fastapi/issues/858>
<https://github.com/tiangolo/fastapi/issues/1713>

**Comment by ycd** [comment]
Yes we have a [github-actions bot](https://github.com/tiangolo/fastapi/blob/master/.github/workflows/issue-manager.yml) for managing & closing the issues, but tbh Stale looks great tho.

**Comment by Kludex** [comment]
The current one is working, but only if the issue was answered by tiangolo or dmontagu.

**Comment by ArcLightSlavik** [comment]
Would need to be triaged and added by @tiangolo or @dmontagu ? 
`Pinging them so they would notice quicker than other issues 🙂 `

**Comment by tiangolo** [comment]
Thanks for the discussion everyone! :coffee: 

Yeah, I know I have to give it a pass through all the open issues :grimacing: 

And it's also unfortunate that it has the name "issues", although it's mainly for questions, there's no better alternative to that for now...

Indeed, I created a GitHub Action to handle these issues in a different way than Stale bot. I don't like the way Stalebot could close issues/questions that were not answered and similar things, you can read the whole motivation here: https://github.com/tiangolo/issue-manager#motivation

But in short, yeah, I have to give it a pass to the current open issues, and label them with `answered` in the cases where some of you (:bow: :nerd_face: :cake: ) already answered the question.
